fibo/test_classes/phase4_UI_tests/test_ui_create_issue.py

```diff
@@ -0,0 +1,37 @@
+import pytest
+from dev_classes.phase4_UI.jira_pages import *
+
+
+class TestCreateIssue:
+    def test_all_required_fields(s, loginToJira):
+        summary = "Scaramouche"
+        main_jira_page: MainPO = loginToJira
+        main_jira_page.open_filter()
+        main_jira_page.create_issue(project + "\n", "Bug", summary)
```

Comment:
Why do you need "\n" at the test level? 
Is this a part of test case data?

Reply:
yes "\n" is a part of method 
its like Keys.ENTER action